### PR TITLE
Decimal functions - rebase

### PR DESCRIPTION
**Pull request by fiedukow**
After rebase to new version of `decimal_coersion` a lot has changed.
Here is old PR: 
https://github.com/Teradata/presto/pull/130

### CODE REVIEW

**Comment by ilfrin** [comment]
why are the two "round" commits?

**Comment by fiedukow** [comment]
I assume that there was a github glitch while force pushing some changes, looks good to me right now.

**Comment by fiedukow** [comment]
merged to sprint-27

File: presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java [hunk 1]
```diff
@@ -143,6 +143,7 @@
 import static com.facebook.presto.operator.aggregation.ArrayAggregationFunction.ARRAY_AGGREGATION;
 import static com.facebook.presto.operator.aggregation.ChecksumAggregationFunction.CHECKSUM_AGGREGATION;
 import static com.facebook.presto.operator.aggregation.CountColumn.COUNT_COLUMN;
+import static com.facebook.presto.operator.aggregation.DecimalAverageAggregation.DECIMAL_AVERAGE_AGGREGATION;
```
Comment (fiedukow) [inline presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java:146]: test

**Pull request closed without merging**